server/src/controllers/Venue/resolver.ts

```diff
@@ -99,9 +101,42 @@ export class VenueResolver {
   @Mutation(() => Venue)
   async deleteVenue(
     @Arg('id', () => Int) id: number,
+    @Arg('chapterId', () => Int) chapterId: number,
     @Arg('_onlyUsedForAuth', () => Int) _onlyUsedForAuth: number,
   ): Promise<{ id: number }> {
     // TODO: handle deletion of non-existent venue
+    const users = await prisma.users.findMany({
+      where: {
+        user_chapters: {
+          every: {
+            chapter_id: chapterId,
+          },
+        },
+        AND: {
+          user_events: {
+            every: {
+              event: {
+                canceled: false,
+                ends_at: { gt: new Date() },
+                venue_type: isPhysical,
+              },
+            },
+          },
+        },
+      },
+    });
```

Comment:
Is the object to find the users who are going to events whose venues have changed? If so, `user_chapters` can be ignored.

Reply:
Good point, we don't have venues linked to multiple chapters, so we don't need to specify the `user_chapters`. 👍

Reply:
Right. If they're not going to an event, they don't need to know the venue has changed :man_shrugging: